chrome/script.browserify.js

```diff
@@ -159,16 +173,67 @@ function getLoginData() {
   );
 }
 
+function usernameToClipboard() {
+  var username = this.split("/");
+  username = username[username.length - 1];
+  toClipboard(username);
+  window.close();
+}
+
+function passwordToClipboard() {
+  chrome.runtime.sendNativeMessage(
+    app,
+    {action: "get", entry: this},
+    function(response) {
+      if(chrome.runtime.lastError) {
+        error = chrome.runtime.lastError.message;
+        m.redraw();
+      }
+
+      chrome.tabs.query(
+        {lastFocusedWindow: true, active: true},
+        function(tabs) {
+          //do not copy login data if URL changed during search.
+          if(tabs[0].url == urlDuringSearch) {
+            toClipboard(response.p);
+            window.close();
+          }
+        }
+      );
+    }
+  )
+}
+
+function toClipboard(s){
+  m.render(
```

Comment:
I would just use plain javascript here, document.createElement - I know from other ticket that firefox treats this `m()` differently than chrome, it doesn't create the element immediately or something like that, maybe that's the reason why it doesn't work in FF

Reply:
even though it works in FF, I still think it's better to use vanilla js in this case